src/v/raft/tests/raft_fixture.cc

```diff
@@ -424,7 +428,11 @@ raft_node_instance::initialise(std::vector<raft::vnode> initial_nodes) {
     co_await _storage.invoke_on_all(&storage::api::start);
     storage::ntp_config ntp_cfg(ntp(), _base_directory);
 
-    auto log = co_await _storage.local().log_mgr().manage(std::move(ntp_cfg));
+    auto log = co_await _storage.local().log_mgr().manage(
+      std::move(ntp_cfg),
+      test_group,
+      _with_offset_translation ? model::offset_translator_batch_types()
+                               : std::vector<model::record_batch_type>{});
```

Comment:
Nit: a bit weird to see an empty vector here. Maybe at least configuration batches, or take the vector as a parameter?